.github/workflows/slack-pr-notifications.yaml

```
@@ -0,0 +1,754 @@
+          {
+            echo "FILE_LIST<<EOF"
+            echo -e "$FILE_LIST"
+            echo "EOF"
+          } >> $GITHUB_ENV
+
+          echo "Detected $SERVICE_COUNT service(s) with changes"
+          echo "File changes: +$FILES_ADDED -$FILES_REMOVED ~$FILES_MODIFIED (Total: $TOTAL_FILES)"
+
+      - name: Determine event type and color
+        id: event-info
+        run: |
```

actionlint (1.7.9)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)